storyblok/storyblok-design-system
# fix(ON-947): app sidebar icon

### this pr shows adds/shows an icon to an SideBar App


## Pull request type

Jira Link: [ON-947](https://storyblok.atlassian.net/browse/ON-947)




## How to test this PR
- Locally, Please checkout to this branch on storyblok-design-system and to this [branch on storyfront repo](https://github.com/storyblok/storyfront/pull/3251)
- Create a new App side bar and add an icon to it
- 
![Captura de ecrã 2023-01-12, às 14 51 50](https://user-images.githubusercontent.com/94900845/212098745-f382f534-518c-4ddd-b346-775c3d381cb6.png)

- go to a Space and check that the sidebar apps now have a default icon if there is no image added or it shows a image if it has
![Captura de ecrã 2023-01-12, às 14 55 12](https://user-images.githubusercontent.com/94900845/212099725-35091341-06bf-4468-886f-6b5eada184f2.png)


- check that the design was slightly updated has seen on figma 
<img width="550" alt="Captura de ecrã 2023-01-12, às 14 54 17" src="https://user-images.githubusercontent.com/94900845/212099394-4f6d57c0-d980-4aa0-b255-b8eb63d5d6c0.png">


## What is the new behavior?



- 
- 
- 

## Other information


[ON-947]: https://storyblok.atlassian.net/browse/ON-947?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
